### PR TITLE
Adding support for status & scale subresources in CRD

### PR DESCRIPTION
**Pull request by vineeth-thumma**

### CODE REVIEW

**Review by vineeth-thumma — COMMENTED** [review]
Please review and merge

**Comment by ant31** [comment]
cc @brancz

File: pkg/cli-utils.go [hunk 1]
```diff
@@ -100,6 +103,15 @@ func NewCustomResourceDefinition(config Config) *extensionsobj.CustomResourceDef
 				Categories: config.Categories,
 				ShortNames: config.ShortNames,
 			},
+			Subresources: &extensionsobj.CustomResourceSubresources{
+				Status: &extensionsobj.CustomResourceSubresourceStatus {
+				},
+				Scale: &extensionsobj.CustomResourceSubresourceScale {
+					SpecReplicasPath:	".spec.replicas",
+					StatusReplicasPath:	".status.replicas",
```
Comment (ant31) [inline pkg/cli-utils.go]: why paths are hardcoded?

Comment (vineeth-thumma) [reply pkg/cli-utils.go]: It's suggested to have the default paths. I've made the changes.

**Review by vineeth-thumma — COMMENTED** [review]
made the changes. Please review @ant31  @brancz

**Comment by brancz** [comment]
lgtm

**Comment by vineeth-thumma** [comment]
@ant31  can you plz review & merge

**Pull request merged**